.gitignore

```diff
@@ -5,3 +5,4 @@ behat.yml
 vendor
 composer.lock
 .phpunit.result.cache
+phpstan.neon
```

Comment:
Allows having a local phpstan.neon files that overrides some rules or modifies some settings

Reply:
I always failed to see the use case for that. Running phpstan locally should use the same config than the CI IMO

Reply:
I have this configuration in this file:
```
includes:
	- phpstan.dist.neon

parameters:
	editorUrl: 'phpstorm://open?file=%%file%%&line=%%line%%'
```
This allows me to open phpstan found errors in phpstorm, but obviously this config is local to me

Reply:
ah, I did not know that phpstan had this parameter. This is indeed a valid use case.

Reply:
Very useful tip, I didn't know that either - thanks!